theme/scripts/cart.ts

```diff
@@ -285,9 +285,10 @@ export default(function() {
       })
     },
 
-    updateProductInfoBar(variantId: string, action: Action) {
+    //TO-DO: This function should check when the add to cart button is clicked and use a setTimeout to change the text to "Added to cart" and then revert back to "Add to cart" after a few seconds.
+    handleAddToCartButton(variantId: string, action: Action) {
       const addToCartButton = document.querySelector(Cart.SELECTORS.addToCartButton);
-
+      
```

Comment:
Added a to-do to update this function. This was written when products only had one variant.
- When the user adds an item to their cart, if the item in the cart has the same `variantId` as the product page you are currently on, the `Add to Cart button` changes to "Added to cart"
- This no longer works now that we have products with several variants. 
- In a future PR, I will update this function to check when the `Add to cart button` is clicked and use a `setTimeout` to change the text to "Added to cart" for a few seconds before reverting back to "Add to cart" so users can add multiple variants to their cart.